submissions/krailoartur/HTML_CSS_practice/style.css

```diff
@@ -0,0 +1,186 @@
+*:focus {
+    outline-offset: 3px;
+    outline: 1px solid #74b9ff;
+}
+
+body {
+    color: #636e72;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+nav ul {
+    height: 70px;
+    margin: 0 0 0 0;
+    padding: 0;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+nav li {
+    margin: 0;
+    padding: 0 14px;
+    list-style: none;
+}
+
+#popup-checkbox {
```

Comment:
Watch  [video](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev) how to hide checkboxes in right way